src/wp-includes/blocks/navigation-submenu.php

```diff
@@ -183,7 +183,16 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 	if ( ! $open_on_click ) {
 		$item_url = isset( $attributes['url'] ) ? $attributes['url'] : '';
 		// Start appending HTML attributes to anchor tag.
-		$html .= '<a class="wp-block-navigation-item__content" href="' . esc_url( $item_url ) . '"';
+		$html .= '<a class="wp-block-navigation-item__content"';
+
+		// The href attribute on a and area elements is not required;
```

Comment:
This should use the correct [multi-line comment format](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments).